docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -80,3 +80,22 @@ the client’s certificate.
 `xpack.monitoring.elasticsearch.ssl.keystore.password`::
 
 Optional settings that provide the password to the keystore.
+
+[[monitoring-additional-settings]]
+===== Additional settings
+
+`xpack.monitoring.elasticsearch.cloud_id`::
+
+If you're using Elasticsearch cloud, you should specify the identifier here.
```

Comment:
Similar to other comment: `s/Elasticsearch cloud/{es} in Elastic Cloud/g`

```suggestion
If you're using {es} in Elastic Cloud, you should specify the identifier here.
```

Reply:
Should be {es} in {ecloud}